src/Groupper.ts

```diff
@@ -166,35 +169,46 @@ export class Groupper
         ) {
             return {
                 element: groupperFirstFocusable,
-                lastMoverOrGroupper: groupperFirstFocusable ? undefined : this,
+                outOfDOMOrder: true,
             };
         }
 
         const tabster = this._tabster;
         let next: HTMLElement | null | undefined = null;
         let uncontrolled: HTMLElement | undefined;
+        let outOfDOMOrder = false;
         const onUncontrolled = (el: HTMLElement) => {
             uncontrolled = el;
         };
 
         if (this._shouldTabInside && groupperFirstFocusable) {
+            let findProps: Types.FindNextProps;
+
             next = isBackward
-                ? tabster.focusable.findPrev({
-                      container: groupperElement,
-                      currentElement,
-                      onUncontrolled,
-                      ignoreUncontrolled,
-                      ignoreAccessibility,
-                      useActiveModalizer: true,
-                  })
-                : tabster.focusable.findNext({
-                      container: groupperElement,
-                      currentElement,
-                      onUncontrolled,
-                      ignoreUncontrolled,
-                      ignoreAccessibility,
-                      useActiveModalizer: true,
-                  });
+                ? tabster.focusable.findPrev(
+                      (findProps = {
+                          container: groupperElement,
+                          currentElement,
+                          referenceElement,
+                          onUncontrolled,
+                          ignoreUncontrolled,
+                          ignoreAccessibility,
+                          useActiveModalizer: true,
+                      })
```

Comment:
Can we re-write this assignments to something more readable?! usually using assignments as expressions is a bit convoluted.

Something like:
```ts
if (this._shouldTabInside && groupperFirstFocusable) {
  const findProps: Types.FindNextProps = {
    container: groupperElement,
    currentElement,
    referenceElement,
    onUncontrolled,
    ignoreUncontrolled,
    ignoreAccessibility,
    useActiveModalizer: true,
  };

  if (isBackward) {
    next = tabster.focusable.findPrev(findProps);
  } else {
    next = tabster.focusable.findNext(findProps);
  }

  outOfDOMOrder = !!findProps.outOfDOMOrderResult;

  if (!uncontrolled && !next && !outOfDOMOrder) {
    next = tabster.focusable.findNext({
      container: groupperElement,
      currentElement: null,
      referenceElement,
      onUncontrolled,
      ignoreUncontrolled,
      ignoreAccessibility,
      useActiveModalizer: true,
    });

    outOfDOMOrder = true;
  }
}
```